tests/python/binding_tests.py

```diff
@@ -499,32 +499,38 @@ def test_deploy_evm():
     assert res == True
 
 
-def test_verify_evm():
```

Comment:
can we add this test back in -- it seems like we no longer test verification inside of python here

Reply:
Sure, done. Sorry, that was totally unintentional.